metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -790,6 +798,59 @@ def get_dataset_properties() -> DatasetPropertiesClass:
                 tags=[],
             )
 
+        def get_s3_tags() -> Optional[GlobalTagsClass]:
+            bucket_name = s3_util.get_bucket_name(
+                table["StorageDescriptor"]["Location"]
+            )
+            tags_to_add = []
+            if self.source_config.use_s3_bucket_tags:
+                bucket_tags = self.s3_client.get_bucket_tagging(Bucket=bucket_name)
+                tags_to_add.extend(
+                    [
+                        make_tag_urn(f"""{tag["Key"]}:{tag["Value"]}""")
+                        for tag in bucket_tags["TagSet"]
+                    ]
+                )
+            if self.source_config.use_s3_object_tags:
+                key_prefix = s3_util.get_key_prefix(
+                    table["StorageDescriptor"]["Location"]
+                )
+                object_tagging = self.s3_client.get_object_tagging(
```

Comment:
What if there are no object tags? Will it return anything or it will return None?